src/Homework3/06_uporyado4_list.py

```diff
@@ -0,0 +1,15 @@
+"""
+Упорядоченный список.
+Дан список целых чисел. Требуется переместить все
+ненулевые элементы в левую часть списка, не меняя
+их порядок, а все нули - в правую часть. Порядок
+ненулевых элементов изменять нельзя, дополнительный
+список использовать нельзя, задачу нужно выполнить
+за один проход по списку. Распечатайте полученный список.
+"""
+
+lst = [0, 1, 0, 1, 2, 0, 7]
+for el in lst:
+    if el == 0:
+        lst.append(lst.pop(lst.index(el)))
```

Comment:
index is making additional for-loop and it contradicts with requirement to make one pass